### ISSUE TITLE
Danger signs inappropriately displaying for young infants

### ISSUE THREAD

**Issue by gillianlevine** [WorldHealthOrganization/smart-emcare#70]
No Danger signs should be displayed or available for young infants. All data elements in this section should be skipped. However, currently, all danger signs are being prompted and mandatory to respond to. apkversion1.0.0_dev and build 1.0.1.rc12.build.130

**Comment by delcroip** [comment]
@gillianlevine, @kaledip  how can we capture the "Convulsing Now" described in symptoms <2 m ? 

@ppanchal-argus Please skip dangersign for <2 in the next release

**Comment by kaledip** [comment]
"Convulsing now" question is applicable for both age groups. So we should ask this question. Stabilised and continue consutation will take us to next questionnaire.
"Convulsion in this illness" should be asked in Symptoms. @gillianlevine pls confirm

**Comment by gillianlevine** [comment]
Correct. But "convulsing now" should not be called a "danger sign" it should be called Signs of Life Threatening Illness (as should Obstructed or absent breathing)-----"kaledip" ***@***.***> wrote: -----To: "WorldHealthOrganization/smart-emcare" ***@***.***>From: "kaledip" ***@***.***>Date: 03/08/2023 09:33AMCc: "gillianlevine" ***@***.***>, "Mention" ***@***.***>Subject: Re: [WorldHealthOrganization/smart-emcare] Danger signs inappropriately displaying for young infants (Issue #70)
"Convulsing now" question is applicable for both age groups. So we should ask this question. Stabilised and continue consutation will take us to next questionnaire.
"Convulsion in this illness" should be asked in Symptoms. @gillianlevine pls confirm

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
--------------------------------------------------------------------------This email and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. If you have received this email in error, please notify us immediately by reply e-mail and delete this message from your system.--------------------------------------------------------------------------
